server/lib/notifications.js

```diff
@@ -1,48 +1,71 @@
 const axios = require('axios');
+const config = require('config');
+
 const activitiesLib = require('../lib/activities');
 const slackLib = require('./slack');
-
-const ACTIVITY_ALL = 'all';
+const activityType = require('../constants/activities');
 
 module.exports = (Sequelize, activity) => {
-  if(!activity.GroupId || !activity.type) {
-    return activity;
-  }
-  return Sequelize.models.Notification.findAll({
-    where: {
-      type: [
-        ACTIVITY_ALL,
-        activity.type
-      ],
-      GroupId: activity.GroupId,
-      // for now, only handle gitter and slack webhooks in this lib
-      // TODO sdubois: move email + internal slack channels to this lib
-      channel: ['gitter', 'slack'],
-      active: true
-    }
-  }).then(notifConfigs =>
-    Promise.all(notifConfigs, notifConfig => {
-      if (notifConfig.channel === 'gitter') {
-        return publishToGitter(activity, notifConfig);
-      } else if (notifConfig.channel === 'slack') {
-        return publishToSlack(activity, notifConfig);
+  // publish everything to our private channel
+  return publishToSlackPrivateChannel(activity,
```

Comment:
If I understood correctly, just noting this will add quite some redundancy, basically we'll get most events twice in public and private channel. Would be nice to just publish to private channel what's not published to the public one.